### DISCUSSION TITLE
Arduino IDE not importing zip file

### DISCUSSION THREAD

**Discussion by JPMulder** [pschatzmann/arduino-audio-tools#990] [Q&A] [answered]
Hi

I think there is an issue with the zip file, 
Cant add to Arduino as zip file and can't extract it either

**Comment by pschatzmann** [comment, marked as answer]
This has nothing to do with my library and I can't provide any basic Arduino or Github Support

Maybe you just try the way that is recommended in the Readme...

**Reply by JPMulder** [reply]
If I download the zip file it says unexpected end of archive, tried to download it a few times already.
Can you kindly try to download and see if it works. Sorry if I'm wasting your time, I'm not asking for Adruino or github support, I'm letting you know that there is no working library to download at the moment

**Comment by pschatzmann** [comment]
Actually you do ask me for github support: for me it is currently working in linux but not on OSX!
